pkgs/development/compilers/jetbrains-jdk/default.nix

```diff
@@ -37,16 +37,16 @@ let
   }.${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
   cpu = stdenv.hostPlatform.parsed.cpu.name;
 in
-openjdk17.overrideAttrs (oldAttrs: rec {
+openjdk21.overrideAttrs (oldAttrs: rec {
   pname = "jetbrains-jdk" + lib.optionalString withJcef "-jcef";
-  javaVersion = "17.0.8";
-  build = "1000.8";
+  javaVersion = "21.0.2";
+  build = "375.1";
```

Comment:
As far as I know, all of the current releases of JetBrains IDEs ship with JetBrains JDK version 17. Even though JetBrains publishes v21 builds of the runtime, using that with the IDEs right _may_ be asking for trouble.

Reply:
Hmm, good point. I'm not too in touch with the JetBrains IDEs ever since I dropped PyCharm ages ago, so I'm not an expert on that. I'll try it out to see if it works, but I'll probably change the PR to just bump to the latest 17 version.